worker/worker.py

```diff
@@ -232,6 +234,9 @@ def main():
   success = True
   global ALIVE
   while ALIVE:
+    print(path.join(worker_dir, 'fish.exit'))
```

Comment:
Should this print be removed or is it an aid for stopping the worker (so user knows where to place 'fish.exit' ?)

Reply:
@tomtor I wrote that print for debug